src/PopupDialog.js

```diff
@@ -1,54 +1,51 @@
 // flow
 
 import React, { PropTypes, Component } from 'react';
-import { Dimensions } from 'react-native';
 import Dialog from './components/Dialog';
 import DialogTitle from './components/DialogTitle';
 import ScaleAnimation from './animations/ScaleAnimation';
 
-const SCRREN_WIDTH = Dimensions.get('window').width;
-const SCRREN_HEIGHT = Dimensions.get('window').height;
-
 const propTypes = {
-  ...Dialog.propTypes,
-  ...DialogTitle.propTypes,
+	...Dialog.propTypes,
+	...DialogTitle.propTypes,
 };
 
 const defaultProps = {
-  animation: 'scale',
-  animationDuration: 200,
-  closeOnTouchOutside: true,
-  dialogAnimation: new ScaleAnimation(),
+	animation: 'scale',
+	animationDuration: 200,
+	closeOnTouchOutside: true,
+	dialogAnimation: new ScaleAnimation(),
 };
 
 class PopupDialog extends Component {
-  static propTypes = propTypes
-  static defaultProps = defaultProps
-
-  openDialog(onOpened) {
-    this.dialog.open(onOpened);
-  }
-
-  closeDialog(onClosed) {
-    this.dialog.closed(onClosed);
-  }
-
-  render() {
-    let title;
-
-    if (this.props.title) {
-      title = <DialogTitle {...this.props} />;
-    }
-
-    return (
-      <Dialog
-        ref={(dialog) => { this.dialog = dialog; }}
-        {...this.props}
-      >
-        {title}
-      </Dialog>
-    );
-  }
+	static propTypes = propTypes;
+	static defaultProps = defaultProps;
+	
+	constructor(props) {
+		super(props);
+	}
+	
+	openDialog(onOpened) {
+		this.dialog.open(onOpened);
+	}
+	
+	closeDialog(onClosed) {
+		this.dialog.closed(onClosed);
+	}
+	
+	render() {
+		const title = this.props.title ? <DialogTitle {...this.props} /> : null;
+		
+		return (
+				<Dialog
+						ref={dialog => this.dialog = dialog}
+						{...this.props}
+				>
+					{title}
+					{this.props.children} // Critical!
```

Comment:
@aviadhahami 
Need remove `// Critical!`, that will make App crash while open dialog.